webapp/components/survey/NodeDefsSelector/AttributesSelector/AttributeSelector.js

```diff
@@ -35,6 +35,7 @@ const AttributeSelector = (props) => {
         className={`btn btn-s deselectable attribute-selector${isActive ? ' active' : ''}`}
         onClick={() => onToggleAttribute(nodeDefUuid)}
         disabled={!canSelectAttributes}
+        title={NodeDef.getLabelWithType({ nodeDef, lang, type: NodeDef.NodeDefLabelTypes.name })}
```

Comment:
maybe it was better to use just NodeDef.getName(nodeDef) in this case...!

Reply:
true because if wee needs to make the change the cost of the change is going to be the same, thanks!